cmake/modules/zephyr_default.cmake

```diff
@@ -31,6 +31,18 @@ find_package(ZephyrBuildConfiguration
   NO_CMAKE_SYSTEM_PACKAGE_REGISTRY
 )
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.22.2)
```

Comment:
"3.22.1 and 3.22.2 are broken" -- https://gitlab.kitware.com/cmake/cmake/-/issues/23187

Reply:
Nice catch. I only noticed the `3.22.2`.